EIPS/eip-969.md

```diff
@@ -0,0 +1,167 @@
+---
+eip: 969 
+title: Modifications to ethash to invalidate dedicated hardware implementations 
+author: David Stanfill <david@airsquirrels.com> 
+discussions-to: https://gitter.im/ethereum/topics/topic/5ac4d974109bb043328911ce/eip-969-discussion 
+status: Draft
+type: Standards Track
+category: Core
+created: 2018-04-03
+---
+
+
+## Simple Summary
+
+This EIP modifies ethash in order to break ASIC miners specialized for the current ethash
+mining algorithm.
+
+
+## Abstract
+
+There are companies who currently have dedicated hardware based ethereum miners in
+production, and may be actively mining.  This EIP aims to "poison
+the well" by modifying the block mining algorithm in a low risk manner that
+may *"break"* these miners if they are in-fact built as traditional ASICs.
+
+
+## Motivation
+
+ASIC based miners will have lower operational costs than GPU based miners which
+will result in GPU based mining quickly becoming unprofitable.  Given that
+production of ASIC based miners has a high barrier to entry and few market players,
+this will cause a trend towards centralization of mining power.  
+
+Risks include market dominance by a single manufacturer that may utilize production 
+stock to mine themselves, introduce backdoors in their hardware, or otherwise 
+facilitate a 51% attack that would be infeasible. 
+
+This trend towards centralization has a negative effect on network security,
+putting significant control of the network in the hands of only a few entities.
+
+Ethash remains ASIC resistant, however ASIC manufacturer technology is advancing
+and ethash may require further changes in order to remain resistant to unforeseen
+design techniques. This EIP seeks explicitly to buy time during which newly developed
+ASIC technology will face a barrier while more long term mechanisms to ensure 
+continued ASIC resistance can be explored.  
+
+## Specification
+
+If `block.number >= ASIC_MITIGATION_FORK_BLKNUM`, require that the ethash solution 
+sealing the block has been mined using `ethashV2`.
+
+## EthashV2
+
+`ethashV2` will be identical in specification to the current `ethash`(v1) algorithm, with
+the exception of the implementation of `fnv`. 
+
+The new algorithm replaces the 5 current uses of `fnv` inside `hashimoto` with 5 
+seperate instances defined as `fnvA`, `fnvB`, `fnvC`, `fnvD`, and `fnvE`, utilizing 
```

Comment:
seperate -> separate

Reply:
Could you also give more information, e.g. as per https://github.com/ethereum/EIPs/issues/958#issuecomment-378037230?

Reply:
Can you clarify the more information from #958? A lot of 958 discussion is now deleted.